README.adoc

```diff
@@ -179,13 +179,13 @@ name-deployment-694c7b74f7-lrlf7   0/1       Running   0          1m
 ping-deployment-cf8f564c6-nctcr    1/1       Running   0          1m
 ----
 
-You will notice that one of the two `name` pods is no longer in the ready state. Navigate to `{name-api}`. Observe that your request will still be successful because you have two replicas and one is still healthy.
+You will notice that one of the two `name` pods is no longer in the ready state. Navigate to `{name-api}`. Observe that your request is still be successful because you have two replicas and one is still healthy.
 
 === Observing the effects on the ping microservice
 
-Wait until the `name` pod is ready again. Make two POST requests to `{name-api}/unhealthy`. If you see the same pod name twice, make the request again until you see that the second pod has been made unhealthy. You may see the same pod twice because there's a delay between a pod becoming unhealthy and the readiness probe noticing it. Therefore, traffic may still be routed to the unhealthy service for approximately 5 seconds. Continue to observe the output of `kubectl get pods`. You will see both pods are no longer ready. During this process, the readiness probe for the `ping` microservice will also fail. Observe it's no longer in the ready state either.
+Wait until the `name` pod is ready again. Make two POST requests to `{name-api}/unhealthy`. If you see the same pod name twice, make the request again until you see that the second pod has been made unhealthy. You might see the same pod twice because there's a delay between a pod becoming unhealthy and the readiness probe noticing it. Therefore, traffic might still be routed to the unhealthy service for approximately 5 seconds. Continue to observe the output of `kubectl get pods`. You will see both pods are no longer ready. During this process, the readiness probe for the `ping` microservice will also fail. Observe that it's no longer in the ready state either.
 
-First, both `name` pods will no longer be ready because the readiness probe failed.
+First, both `name` pods are no longer be ready because the readiness probe failed.
```

Comment:
```suggestion
 First, both `name` pods are no longer ready because the readiness probe failed.
```

Reply:
> First, both `name` pods are no longer be ready because the readiness probe failed.

should be

> First, both `name` pods are no longer ready because the readiness probe failed.